src/components/Layout/RightPaneLayout/RightPaneLayout.tsx

```diff
@@ -0,0 +1,51 @@
+import { AnimatePresence, motion } from "framer-motion";
+import React, { ReactElement } from "react";
+import { Css, Palette, px } from "../../../Css";
+import { useRightPaneContext } from "./RightPaneContext";
+
+export function RightPaneLayout({
+  children,
+  paneBgColor = Palette.White,
+  paneWidth = 450,
+}: {
+  children: ReactElement;
+  paneBgColor?: Palette;
+  paneWidth?: number;
+}) {
+  const { isRightPaneOpen, rightPaneContent, clearPane } = useRightPaneContext();
+
+  return (
+    <div css={Css.h100.df.overflowXHidden.$}>
+      <>
+        <div
+          css={
+            Css.add("width", `calc(100% - ${paneWidth + 24}px)`)
```

Comment:
**nit (non-blocking):** This could be `Css.w(`calc(100% - ${paneWidth + 24}px)`).$`. (no need of the `Css.add`)